STM32/Libraries/Util/Src/CAProtocol.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * CAProtocol.c
+ *
+ *  Created on: Oct 6, 2021
+ *      Author: agp
+ */
+#include <stdio.h>
+#include <string.h>
+#include <strings.h>
+#include <stdlib.h>
+#include "CAProtocol.h"
+
+typedef struct CAProtocolData {
+    // Nothing for now.
+} CAProtocolData;
+
+#define MAX_NO_CALIBRATION 12
+static void calibration(CAProtocolCtx* ctx, const char* input)
```

Comment:
Great with a generic function for calibration! :) 
I see a potential "disaster" if alpha and beta are reversed on the input. Do we have any way of ensuring the validity of the incoming data?  
Some easy/understandable user interface in LoopControl maybe - at least we should be aware of some potential errors.